docs/rest/Data/CompositeCustomSearchParameter.json

```diff
@@ -0,0 +1,42 @@
+{
+    "resourceType" : "SearchParameter",
+    "id" : "Patient-address-use-postcode",
+    "extension" : [{
+      "url" : "http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status",
+      "valueCode" : "trial-use"
+    }],
+    "url" : "http://hl7.org/fhir/SearchParameter/Patient-address-use-postcode",
+    "version" : "1.0.0",
+    "name" : "address-use-postcode",
+    "status" : "draft",
+    "experimental" : false,
+    "date" : "2019-11-01T09:29:23+11:00",
+    "publisher" : "Health Level Seven International (Orders and Observations)",
+    "contact" : [{
+      "telecom" : [{
+        "system" : "url",
+        "value" : "http://hl7.org/fhir"
+      }]
+    },
+    {
+      "telecom" : [{
+        "system" : "url",
+        "value" : "http://www.hl7.org/Special/committees/orders/index.cfm"
+      }]
+    }],
+    "description" : "Address use and postalCode parameter pair",
+    "code" : "address-use-postcode",
+    "base" : ["Patient"],
+    "type" : "composite",
+    "expression" : "Patient",
+    "xpathUsage" : "normal",
+    "multipleOr" : false,
+    "component" : [{
+      "definition" : "http://hl7.org/fhir/SearchParameter/individual-address-use",
+      "expression" : "address.use"
+    },
```

Comment:
Nit: this composite search parameter can't be parsed properly if we are running STU3 - maybe we could add a comment at the top of the `.http` test file to point people to using R4 or R5?

Reply:
Comment added in http file